integration-tests/search.test.ts

```diff
@@ -570,7 +570,13 @@ describe('test searches with different scopes', () => {
 
             const getPatient = await client.get('Patient');
             expect(getPatient.data.total).toBe(2); // Should return Mycroft Holmes as a seealso
-            expect(getPatient.data.entry[1].resource).toMatchObject(sherlockHolmes);
+            const entrySherlockResource = getPatient.data.entry.find(
+                (resource: { resource: { id: string } }) => resource.resource.id === idsOfFhirResources.sherlockHolmes,
+            );
+            if (!entrySherlockResource) {
```

Comment:
Check if entrySherlockResource match object sherlockHolmes as well?

Reply:
Yup that's done [here](https://github.com/awslabs/fhir-works-on-aws-deployment/pull/699/files#diff-d9f1abce6e1fbd388039799e21b9cb38e704853f086342bde826bbd961291c98R579)